tests/entrypoints/openai/test_accuracy.py

```diff
@@ -18,12 +20,21 @@
 FILTER = "exact_match,strict-match"
 RTOL = 0.03
 EXPECTED_VALUE = 0.58
-DEFAULT_ARGS = ["--max-model-len", "4096", "--disable-log-requests"]
+DEFAULT_ARGS = ["--max-model-len", "2048", "--disable-log-requests"]
```

Comment:
You've got a change here that will be in effect for other users of `test_accuracy.py`. That would be `run-amd-test.sh` at least. Is that intentional? I just ask since it wasn't called out in the commit message / PR description.

Reply:
I just wanted to make sure that things will fit in whatever system we run it on. So going from 4096 -> 2048 should not impact anything beyond fitting in systems with smaller ram since all the gsm prompts are much shorter than this. Did you see a failure somewhere?

Reply:
updated comment

Reply:
no problems - just making sure it was accidentally left over since it wasn't TPU specific

Reply:
2048 should be fine for gsm8k evals